packages/material-ui/src/Tab/Tab.js

```diff
@@ -36,7 +36,7 @@ export const styles = theme => ({
       marginBottom: 6,
     },
   },
-  /* Styles applied to the root element if `textColor="inherit"`. */
+  /* Styles applied to the root element if `<Tabs textColor="inherit" />`. */
```

Comment:
> The prop is private on the Tab level, it's public at the Tabs level. The classes are public.

Oh right that is even more confusing now. We're talking about `TabS` in the `Tab` API. The current sentence can be easily misread as `<Tab textColor />`.

```suggestion
  /* Styles applied to the root element the parent [`Tabs`](/api/tabs/) `textColor="inherit"`. */
```

Reply:
looks better!